examples/charts/rendering-spec/package.json

```
@@ -0,0 +1,24 @@
+{
+  "name": "charts-sdk-example-rendering-spec",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.html",
+  "scripts": {
+    "start": "parcel index.html --open",
+    "build": "parcel build index.html"
+  },
+  "dependencies": {
+    "@alenaksu/json-viewer": "^2.0.1",
+    "@mongodb-js/charts-embed-dom": "^3.0.0",
```

```suggestion
    "@mongodb-js/charts-embed-dom": "^3.2.0",
```

Since v3.0.0 won't work with this, let's force users to update to the new version. I understand that this does not exist yet so we can wait until then (when we update the link too).

---

Yup, this is the "next steps" section points :)